### PR TITLE
Update documentation to remove docker hub references

### PR DESCRIPTION
**Pull request by williamsbritt**
Signed-off-by: “Brittany <brittany.c.williams@ibm.com>

### CODE REVIEW

**Comment by Domenick97** [comment]
Theres a reference to `DOCKERHUB_DOMAIN := ibmcom` in the Makefile. Not sure if this should be removed or not

File: docs/developer-tool-faq.md [hunk 1]
```diff
@@ -523,15 +523,15 @@ git config --global core.safecrlf false
...
 1. If you are using a latest tag such as `latest`.
-    1. In a terminal, run `docker pull ibmcom/detect-secrets-hook:latest` to get the latest image.
+    1. In a terminal, run `docker pull icr.io/git-defenders/detect-secrets-hook:latest` to get the latest image.
 1. If you are using a specific version tag, such as `0.13.1+ibm.37.dss`
     1. Update the `.pre-commit-config.yaml` to use a newer version tag.
 
 ### Can I pull a specific version of docker image?
 
-Yes, any tag listed [in docker hub for image ibmcom/detect-secrets](https://hub.docker.com/repository/docker/ibmcom/detect-secrets) can be used. You can use the same approach to find tags for `ibmcom/detect-secrets-hook`.
+Yes, any tag listed [in docker hub for image icr.io/git-defenders/detect-secrets](https://hub.docker.com/repository/docker/icr.io/git-defenders/detect-secrets) can be used. You can use the same approach to find tags for `icr.io/git-defenders/detect-secrets-hook`.
```
Comment (Domenick97) [inline docs/developer-tool-faq.md:534]: I don't think thats the correct link

**Review by Domenick97 — CHANGES_REQUESTED** [review]

File: docs/developer-tool-faq.md (hunk 1 above)
Comment (victoria-miltcheva) [reply docs/developer-tool-faq.md:534]: Ah never mind, you're talking about https://hub.docker.com/repository/docker/icr.io/git-defenders/detect-secrets 🤦, my bad. Users don't have access to a GUI for viewing the IBM Cloud Container Registry images. They would have to use the IBM Cloud CLI Container Registry plugin to list tags.

Comment (williamsbritt) [reply docs/developer-tool-faq.md:534]: ahh when I did the changes the first time I had the change right but I forgot the second time through- will fix

File: docs/developer-tool-faq.md [hunk 2]
```diff
@@ -523,15 +523,15 @@ git config --global core.safecrlf false
...
 1. If you are using a latest tag such as `latest`.
-    1. In a terminal, run `docker pull ibmcom/detect-secrets-hook:latest` to get the latest image.
+    1. In a terminal, run `docker pull icr.io/git-defenders/detect-secrets-hook:latest` to get the latest image.
 1. If you are using a specific version tag, such as `0.13.1+ibm.37.dss`
     1. Update the `.pre-commit-config.yaml` to use a newer version tag.
 
 ### Can I pull a specific version of docker image?
 
-Yes, any tag listed [in docker hub for image ibmcom/detect-secrets](https://hub.docker.com/repository/docker/ibmcom/detect-secrets) can be used. You can use the same approach to find tags for `ibmcom/detect-secrets-hook`.
+Yes, any tag listed in ICR accessed via the [IBM Cloud Container Registry CLI](https://cloud.ibm.com/docs/Registry?topic=container-registry-cli-plugin-containerregcli) for image icr.io/git-defenders/detect-secrets can be used. You can use the same approach to find tags for `icr.io/git-defenders/detect-secrets-hook`.
```
Comment (victoria-miltcheva) [inline docs/developer-tool-faq.md:534]:
```suggestion
Yes, any tag listed in the IBM Cloud Container Registry (ICR) accessed via the [IBM Cloud Container Registry CLI](https://cloud.ibm.com/docs/Registry?topic=container-registry-cli-plugin-containerregcli) for image `icr.io/git-defenders/detect-secrets` can be used. You can use the same approach to find tags for `icr.io/git-defenders/detect-secrets-hook`.
```

File: docs/developer-tool-faq.md [hunk 3]
```diff
@@ -523,15 +523,15 @@ git config --global core.safecrlf false
...
 1. If you are using a latest tag such as `latest`.
-    1. In a terminal, run `docker pull ibmcom/detect-secrets-hook:latest` to get the latest image.
+    1. In a terminal, run `docker pull icr.io/git-defenders/detect-secrets-hook:latest` to get the latest image.
 1. If you are using a specific version tag, such as `0.13.1+ibm.37.dss`
     1. Update the `.pre-commit-config.yaml` to use a newer version tag.
 
 ### Can I pull a specific version of docker image?
 
-Yes, any tag listed [in docker hub for image ibmcom/detect-secrets](https://hub.docker.com/repository/docker/ibmcom/detect-secrets) can be used. You can use the same approach to find tags for `ibmcom/detect-secrets-hook`.
+Yes, any tag listed in the IBM Container Registry (ICR) accessed via the [IBM Cloud Container Registry CLI](https://cloud.ibm.com/docs/Registry?topic=container-registry-cli-plugin-containerregcli) for image icr.io/git-defenders/detect-secrets can be used. You can use the same approach to find tags for `icr.io/git-defenders/detect-secrets-hook`.
```
Comment (victoria-miltcheva) [inline docs/developer-tool-faq.md:534]:
```suggestion
Yes, any tag listed in the IBM Container Registry (ICR) accessed via the [IBM Cloud Container Registry CLI](https://cloud.ibm.com/docs/Registry?topic=container-registry-cli-plugin-containerregcli) for image `icr.io/git-defenders/detect-secrets` can be used. You can use the same approach to find tags for `icr.io/git-defenders/detect-secrets-hook`.
```

**Review by victoria-miltcheva — APPROVED** [review]

**Review by Domenick97 — APPROVED** [review]

**Pull request merged**